lib/utils/txhelper.js

```diff
@@ -13,17 +13,17 @@ module.exports = {
       }
     }
     var resultJSON = {
-      hash: to.hex(tx.hash()),
-      nonce: to.hex(tx.nonce),
-      blockHash: to.hex(block.hash()),
-      blockNumber: to.hex(block.header.number),
-      transactionIndex: to.hex(transactionIndex),
-      from: to.hex(tx.from),
-      to: to.hex(tx.to),
-      value: to.hex(tx.value),
-      gas: to.hex(tx.gasLimit),
-      gasPrice: to.hex(tx.gasPrice),
-      input: to.hex(tx.data),
+      hash: to.rpcDataHexString(tx.hash()),
+      nonce: to.rpcDataHexString(tx.nonce),
```

Comment:
Nonce should be a quantity string per https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactionbyhash